app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/notification/VpnEnabledNotificationBuilder.kt

```diff
@@ -50,54 +48,47 @@ class DeviceShieldEnabledNotificationBuilder {
             }
         }
 
-        fun buildDeviceShieldEnabledNotification(
+        fun buildVpnEnabledNotification(
             context: Context,
-            deviceShieldNotification: DeviceShieldNotificationFactory.DeviceShieldNotification,
-            pendingIntent: PendingIntent?,
+            vpnEnabledNotificationContent: VpnEnabledNotificationContentPlugin.VpnEnabledNotificationContent,
         ): Notification {
             registerOngoingNotificationChannel(context)
 
             val notificationLayout = RemoteViews(context.packageName, R.layout.notification_device_shield_enabled)
```

Comment:
NIT, if this is now meant to be generic, will be good to rename the layouts as well.

Reply:
will do